rmw_connext_dynamic_cpp/src/functions.cpp

```diff
@@ -2017,6 +2855,7 @@ rmw_take_response(
     return RMW_RET_ERROR;
   }
 
+  // TODO(wjwwood): return this loan, maybe it's not necessary? I don't see a function to do it.
```

Comment:
The documentation (http://community.rti.com/rti-doc/510/ndds/doc/html/api_cpp/classconnext_1_1Requester.html#aeccadc80d171a7ae6573369e9d9bfbbb) indicates that the loan needs to be returned explicitly using `return_loan` or implicitly by the destructor.